### PR TITLE
 NEWS: Add 1.16.0-rc4 Bugfixes section

### PR DESCRIPTION
**Pull request by tvegas1**
## What
List latest backported bugfixes.

## How
Depends on:
- v1.16.x #9702 
- v1.16x #9705

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -11,6 +11,13 @@
 ### Features:
 ### Bugfixes:
 
+## 1.16.0-rc4 (February 21, 2024)
+### Bugfixes:
+#### UCT
+* Fixed Infiniband posting error when compiler optimizes code with memmove()
```
Comment (brminich) [inline NEWS:17]: afair you did not port that fix to `1.16`

Comment (tvegas1) [reply NEWS:17]: agree, but I am assuming that we will port it, else I would remove that line before merge.

Comment (brminich) [reply NEWS:17]: @yosefe, do we need to port #9692?

Comment (tvegas1) [reply NEWS:17]: created #9705, updated description of this PR

**Review by brminich — APPROVED** [review]

**Review by shasson5 — APPROVED** [review]

**Pull request merged**